tests/rptest/services/kerberos.py

```diff
@@ -147,10 +147,14 @@ def _form_kadmin_cmd(self, cmd):
     def _hard_delete_principals(self, node):
         node.account.ssh(f"rm -fr {KDC_DB_PATH}*", allow_fail=True)
 
+    def start_cmd(self):
+        cmd = f"krb5kdc -P {KRB5KDC_PID_PATH};kadmind -P {KADMIND_PID_PATH}"
```

Comment:
You may want to use `&&` rather than `;`  The `kadmind` will always run regardless of result of `krb5kdc`